egograph/backend/tests/unit/test_dataset_contracts.py

```
@@ -0,0 +1,120 @@
+    fixture_name: str,
+    path_attr: str,
+    request,
+):
+    """fixture parquet に対して契約カラムを使った代表クエリが実行できる。"""
+    # Arrange
+    parquet_path = _fixture_parquet_path(request, fixture_name, path_attr)
+    conn = request.getfixturevalue(fixture_name)
+    columns = ", ".join(f'"{column}"' for column in dataset.column_types)
+
+    # Act / Assert
+    assert conn.execute(f"SELECT count(*) FROM '{parquet_path}'").fetchone()[0] > 0
```

Ruff (0.16.0)

[error] Possible SQL injection vector through string-based query construction

(S608)